gui/mainwindow.cpp

```diff
@@ -486,6 +486,11 @@ void MainWindow::saveSettings() const
 
 void MainWindow::doAnalyzeProject(ImportProject p, const bool checkLibrary, const bool checkConfiguration)
 {
+    QSharedPointer<Settings> checkSettingsPtr = getCppcheckSettings();
```

Comment:
For information; I moved this `getCppcheckSettings` call up because if there is a failure and we want to abort it is better that we abort early before `clearResults` and other stuff..